### ISSUE TITLE
isAlive has been deprecated and removed in favor of is_alive in Python 3.9

### ISSUE THREAD

**Issue by tirkarthi** [IBM/ibm-cos-sdk-python-core#10]
isAlive has been deprecated alias for is_alive and has been removed in Python 3.9 . Python 3.9 PR : https://github.com/python/cpython/pull/15225

```
ibm-cos-sdk-python-core/ibm_botocore/credentials.py
2236:                if self._background_thread.isAlive():
2291:                            if not self._background_thread.isAlive():
```

**Comment by IBMeric** [comment]
Thank you for bringing this issue to our attention. While we do not currently support unreleased versions of Python, we do plan to resolve this issue in an upcoming patch release.

**Comment by IBM-Justin** [comment]
We appreciate your feedback and are looking into addressing this enhancement in Q4 of 2020. Please close out this ticket as it is now being tracked on our internal roadmap.

**Comment by IBM-Justin** [comment]
We are closing this ticket now as it is being tracked on our internal enhancement roadmap.

**Comment by IBMeric** [comment]
@tirkarthi This issue has been resolved in the 2.9.0 release.
